content/efcore/getting-started/toc.md

```diff
@@ -1 +1,2 @@
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```

Comment:
I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`